### PR TITLE
refactor(client): change navbar and fab css

### PR DESCRIPTION
**Pull request by nihalanisumit**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [X] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by piyushagwl007** [comment]
@nihalanisumit can you add a preview for this?

**Comment by nihalanisumit** [comment]
nb fab
@piyushagwl007 
Changes
1) Nav bar color
2) nav items hover and active color
3) aligned profile dropdown with other nav items
4) Dropdown menu opens towards left
5) FAB color change

File: client/src/components/ProfileDropdown/ProfileDropdown.jsx [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+  <div className={styles.container}>
+  <Dropdown text="Profile" item floating>
+    <Dropdown.Menu direction={"left"}>
       <Dropdown.Item text="Profile" />
-      <Dropdown.Item text="My blogs" />
       <Dropdown.Item text="Sign out" onClick={onLogout} />
     </Dropdown.Menu>
   </Dropdown>
+  </div>
 );
```
Comment (piyushagwl007) [inline client/src/components/ProfileDropdown/ProfileDropdown.jsx:17]: Why did you move this file?

**Review by piyushagwl007 — APPROVED** [review]

File: client/src/components/ProfileDropdown/ProfileDropdown.jsx (hunk 1 above)
Comment (nihalanisumit) [reply client/src/components/ProfileDropdown/ProfileDropdown.jsx:17]: I like to have css and jsx in a same folder that's why.
